h/session.py

```diff
@@ -76,10 +76,7 @@ def _current_groups(request, authority):
 
     user = request.user
     svc = request.find_service(name='list_groups')
-    if request.feature('filter_groups_by_scope'):
-        groups = svc.request_groups(user=user, authority=authority)
-    else:
-        groups = svc.all_groups(user=user, authority=authority)
+    groups = svc.session_groups(user=user, authority=authority)
```

Comment:
psst...here is the change to make `session` use a different method on the service.

Reply:
Good to see some logic leaving `session.py` here too